tests/Unit/Console/Commands/CacheAnnualStatisticsTest.php

```diff
@@ -41,7 +41,7 @@
         'year'         => $currentYear,
         'transactions' => 5,
         'volume'       => '50.0000000000000000',
-        'fees'         => '0.0000105',
+        'fees'         => '0.0000000000000105',
```

Comment:
these are all very small 🤔 for a regular transfer with default values (21k gas limit, 5 gwei fee) the actual fee is `0.000105` . Is it expected for the tests to be these small values?